ej2-vue/grid/how-to/access-grid-instance.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Access grid instance in Vue Grid component | Syncfusion
-description: Learn here all about Access grid instance in Syncfusion Vue Grid component of Syncfusion Essential JS 2 and more.
+description: Learn here all about access grid instance in Syncfusion Essentials Vue Grid control, its elements and more.
```

Comment:
Update like this - Learn here all about access grid instance in Syncfusion Essential Vue Grid component, its elements, and more.

Reply:
resolved

Reply:
resolved